packages/blue-sdk-ethers-liquidation/src/LiquidationEncoder.ts

```diff
@@ -9,4 +33,238 @@ export class LiquidationEncoder extends ExecutorEncoder {
   get provider() {
     return this.runner.provider!;
   }
+
+  async handlePendleTokens(
+    chainId: ChainId,
+    market: Market,
```

Comment:
It seems only `market.config.collateralToken` is used, maybe worth limiting input to what's actually needed

But I'm not blocking the pr for that